Understanding Ownership/References and Borrowing/Print Months Once Again/task.md

```diff
@@ -4,6 +4,10 @@ Something strange is going on here. The program is compiled without any problems
 
 Refactor this program to avoid array copying by switching to a mutable reference in `fn print_months_reversed` and fix the compiler errors.
 
+<div class="hint">
+  Remember as we discussed in <a href="course://Common Programming Concepts/Tuples and Arrays/The Array Type">The Array Type</a> task, arrays are allocated on the stack.
```

Comment:
Remember, as we discussed in <a href="course://Common Programming Concepts/Tuples and Arrays/The Array Type">The Array Type</a> task, arrays are allocated on the stack.